environment.yml

```diff
@@ -34,7 +46,8 @@ dependencies:
   # used for formatting code within .ipynb files
   - conda-forge::jupyterlab_code_formatter
   - pip:
-      - git+https://github.com/cytomining/pycytominer.git@8e3c28d3b81efd2c241d4c792edfefaa46698115
+      - pycytominer
```

Comment:
I have been told before that it is preferred to install pycytominer from conda-forge instead, so I would recommend making the change from pip.

Reply:
Good point! Let's shift to the conda release.